dialogflow/cloud-client/src/main/java/com/example/dialogflow/IntentManagement.java

```diff
@@ -18,6 +18,7 @@
 
 // Imports the Google Cloud client library
 
+import com.google.cloud.dialogflow.v2.AgentName;
```

Comment:
I don't see any tags in this file - is that an issue?

Reply:
There are multiple tags.
I will need to create CL before I make changes to IntentManagement.java.

I will create different PR which includes splitting IntentMgmt class into seperate files, updating to latest format and etc..